src/computeTargets.js

```diff
@@ -3,7 +3,7 @@ import * as util from "./util.js";
 export default function computeTargets (root = document.documentElement) {
 	let info = {};
 	let start = performance.now();
-	let links = root.querySelectorAll("a[href^='#']:not(.page-number)");
+	let links = root.querySelectorAll("a[href^='#'].page-number");
```

Comment:
This is to avoid the actual page numbers from being picked up, but I see it's also excluding the TOC page numbers, so we need to make it more specific. But we certainly don't want to only target .page-number links!!

Reply:
What if we give page numbers inside TOC a `toc-page-number` class to distinguish them from page numbers in the footer?

Reply:
Why not simply use selectors for this instead of inventing a new class?

Reply:
I'd love to, though I'm not sure what assumptions we can make about the content to modify the selector. I'm going to think about it a bit more.